From d37767b14991f5883fbe01d955b9fad2ef39581a Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Tue, 15 Jan 2019 22:30:35 +0200
Subject: [PATCH 16/17] sepolicy: Allow mm-qcamerad use appdomain fd

* snap_app requires it for Snap and platform_app requires it for Camera2
  so just modify existing rules and allow it for all apps

Change-Id: Ie2a710f45c24567e5bb1161e1edf5a696888fba4
---
 common/mm-qcamerad.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/mm-qcamerad.te b/common/mm-qcamerad.te
index 70e14d1..f6dd5e5 100644
--- a/common/mm-qcamerad.te
+++ b/common/mm-qcamerad.te
@@ -40,7 +40,7 @@ allow mm-qcamerad self:process execmem;
 allow mm-qcamerad video_device:dir r_dir_perms;
 allow mm-qcamerad { gpu_device video_device sensors_device }:chr_file rw_file_perms;
 
-allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera untrusted_app_all priv_app }:fd use;
+allow mm-qcamerad { surfaceflinger mediaserver cameraserver hal_camera appdomain }:fd use;
 
 allow mm-qcamerad camera_data_file:dir w_dir_perms;
 #allow mm-qcamerad camera_data_file:sock_file { create unlink };
-- 
2.17.1

